src/components/utils/src/logger.cc

```diff
@@ -51,6 +51,16 @@ void deinit_logger () {
   logger::logger_status = logger::LoggerThreadNotCreated;
 }
 
+// Don't destroy logger here!
+// It's just for unloading logger queue
+void flush_logger() {
+  logger::LoggerStatus old_status = logger::logger_status;
+  // Stop pushing new messages to the log queue
+  logger::logger_status = logger::DeletingLoggerThread;
```

Comment:
@anosach-luxoft , because only this status dropping new messages and don't create new logger. No any action happens to this status.